web/deeamon128/html/navigation-menu/style.css

```diff
@@ -0,0 +1,185 @@
+:root {
+  --pickled-bluewood: #2B4448;
+  --rolling-stone: #758086;
+  --burnt-sienna: #FFB9B9;
+  --white: #FFFFFF;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: "Nunito"; sans-serif;
+}
+
+.nav-container {
+  width: 100%;
+  background-color: var(--pickled-bluewood);
+}
+
+.navbar {
+  display: grid;
+  grid-template-columns: 0.2fr auto 1fr;
```

Comment:
you can change
`  grid-template-columns: 0.2fr auto 1fr;`

to 

`  grid-template-columns: 1fr auto 5fr;`